pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -34,6 +34,15 @@ def __lt__(self, requirement: 'Requirement') -> int:
         else:
             return self.name < requirement.name
 
+    def is_complete(self) -> bool:
+        return self.value is not None and not self.value.strip().endswith(b'\\')
```

Comment:
this should probably a little more specific, I would do `.rstrip(b'\r\n')` -- that way something that has trailing whitespace but a backslash at the end isn't considered a continuation

Reply:
Right, I missed that. Fixed